packages/thumbnail-grid-block/src/components/Thumbnail/ThumbnailImageWrapper.tsx

```diff
@@ -0,0 +1,22 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { ReactElement } from 'react';
+import { ThumbnailStylesProps } from '../../types';
+
+type ThumbnailImageWrapperProps = {
+    children: ReactElement;
+    thumbnailStyles: ThumbnailStylesProps;
+};
+
+export const ThumbnailImageWrapper = ({ children, thumbnailStyles }: ThumbnailImageWrapperProps) => {
+    const { width, alignmentClassNames, imageIsAboveOrBelow } = thumbnailStyles;
+    return (
+        <div
+            className={alignmentClassNames}
+            style={{ width: imageIsAboveOrBelow ? undefined : width }}
+            data-test-id="thumbnail-image-wrapper"
+        >
+            <div style={{ width: imageIsAboveOrBelow ? width : undefined }}>{children}</div>
```

Comment:
```suggestion
            <div style={{ width: imageIsAboveOrBelow ? width : 'auto' }}>{children}</div>
```

prefer `'auto'` over `undefined`